packages/flutter_nekoton_bridge/lib/nekoton/core/accounts_storage/accounts_storage.dart

```diff
@@ -61,79 +61,79 @@ class AccountsStorage {
+    required Address rootTokenContract,
   }) async {
     final encoded = await accountsStorage.addTokenWallet(
-      accountAddress: accountAddress,
+      accountAddress: address.address,
       networkGroup: networkGroup,
-      rootTokenContract: rootTokenContract,
+      rootTokenContract: rootTokenContract.address,
     );
     final decoded = jsonDecode(encoded) as Map<String, dynamic>;
     await _updateData();
     return AssetsList.fromJson(decoded);
   }
 
   /// Remove token wallet signature from account (remove token from account aka disable it via slider).
-  /// [accountAddress] - address of account
+  /// [address] - address of account
   /// [networkGroup] - name of network group where this token must be visible, could be found in
   ///   connection info
   /// [rootTokenContract] - address of token in blockchain.
   /// Return updated AssetsList or throw error.
   Future<AssetsList> removeTokenWallet({
-    required String accountAddress,
+    required Address address,
     required String networkGroup,
-    required String rootTokenContract,
+    required Address rootTokenContract,
   }) async {
     final encoded = await accountsStorage.removeTokenWallet(
-      accountAddress: accountAddress,
+      accountAddress: address.address,
       networkGroup: networkGroup,
-      rootTokenContract: rootTokenContract,
+      rootTokenContract: rootTokenContract.address,
     );
     final decoded = jsonDecode(encoded) as Map<String, dynamic>;
     await _updateData();
     return AssetsList.fromJson(decoded);
   }
 
   /// Remove account from storage and return true if it was removed or false.
-  /// [accountAddress] - address of account
+  /// [address] - address of account
   /// Return if asset was removed or throw error.
-  Future<bool> removeAccount(String accountAddress) async {
+  Future<bool> removeAccount(Address address) async {
     final encoded = await accountsStorage.removeAccount(
-      accountAddress: accountAddress,
+      accountAddress: address.address,
     );
     if (encoded == null) return false;
     await _updateData();
     return true;
   }
 
   /// Remove list of account addresses that were removed from storage.
-  /// [accountAddresses] - list of addresses of accounts.
+  /// [addresses] - list of addresses of accounts.
   /// Return list of addresses that were removed or throw error.
-  Future<List<String>> removeAccounts(List<String> accountAddresses) async {
+  Future<List<Address>> removeAccounts(List<Address> addresses) async {
     final encoded = await accountsStorage.removeAccounts(
-      accountAddresses: accountAddresses,
-    );
+        accountAddresses: addresses.map((address) => address.address).toList());
     final decoded = jsonDecode(encoded) as List<dynamic>;
     await _updateData();
     return decoded
-        .map((e) => AssetsList.fromJson(e as Map<String, dynamic>).address)
+        .map((e) =>
+            Address(address: e.fromJson(e as Map<String, dynamic>).address))
```

Comment:
This won't work, because `Map` object doesn't have getter address like in JS, you should save `AssetsList.fromJson(e as Map<String, dynamic>).address`

Reply:
*